src/helpers/group/group-helper.js

```diff
@@ -4,7 +4,10 @@ import { getGroupApi } from '../shared/user-login';
 const groupApi = getGroupApi();
 
 export async function fetchGroups({ limit, offset, name, orderBy }) {
-  return await groupApi.listGroups(limit, offset, name, orderBy);
+  return {
+    ...await groupApi.listGroups(limit, offset, name, orderBy),
```

Comment:
is this going to be sequential?

Reply:
Yeah, good ponit! I've changed it to use `Promise.all` so these promises don't wait for each other.